codex/stores/blockstore.nim

```diff
@@ -29,7 +29,9 @@ type
   BlockType* {.pure.} = enum
     Manifest, Block, Both
 
+  CidCallback* = proc(cid: Cid): Future[void] {.gcsafe, upraises:[].}
```

Comment:
`upraises` is deprecated, you should use `raises: []`.

Reply:
easy fix!